doc/example-workflow.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish
```

Comment:
I think this file should be moved to `docs/` instead of doc/, as that seems to be the usual directory name